packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ChangeEvent } from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { id?: string; value: string; label: string; disabled: boolean };
```

Comment:
@yauheni-kryzhyk-deriv  In some places, `RadioGroup.Item`  has a property called `key`. For example:
 - account/src/Components/personal-details/personal-details.jsx
 - packages/trader/src/Modules/Trading/Components/Elements/Multiplier/cancel-deal-mobile.jsx
 - packages/trader/src/Modules/Trading/Containers/Multiplier/multiplier-options-modal.jsx
 
 Dont we need to add it in the props types? Also` id` prop is available in HTMLAttributes. Should we declare it again?

Reply:
@amina-deriv  I believe this 'key' property is used by React during .map() and we should add it to props.
what about id, as here we are passing explicitly id to component, and only then to html element.
I agree, maybe it is redundant, but as I see mostly we used the same approach in our code. (see please loading.tsx, toggle-switch.tsx, modal.tsx, icon.tsx, checkbox.tsx), but we skip it in button.tsx

Reply:
Got it. Thanks for the explanation @yauheni-kryzhyk-deriv